app/services/profiles/update.rb

```diff
@@ -19,14 +17,16 @@ def initialize(user, updated_attributes)
     end
 
     def call
-      if verify_profile_image && update_profile && sync_to_user
+      if update_successful?
         @user.touch(:profile_updated_at)
-        follow_hiring_tag
+        follow_hiring_tag if SiteConfig.dev_to?
```

Comment:
I cringe whenever I see environment specific. This feels like something simple that we could get rid of to help unify the behavior of all of the Forems and reduce "one-off"'s and complexity of the code. 

Rather than having the attribute, "looking_for_work", Im thinking we remove that and instead ask if they would like to follow the hiring tag. Using a user attribute to then follow a tag seems a little roundabout to me. While I don't think we need to do it in this PR, I think adding a TODO here to follow up with that cleanup would be super helpful and be a nice code cleanup.

Reply:
This is existing behavior that was moved from `UsersController` to this service object. I wasn't sure if we want to remove it from DEV just yet, hence the conditional.

> Im thinking we remove that and instead ask if they would like to follow the hiring tag.

We can't assume this tag will exist across Forems. Or do you mean only on DEV?

> Using a user attribute to then follow a tag seems a little roundabout to me. 

Agreed, but like I said, I just moved the existing implementation.

Reply:
I added a comment now. Please let me know if you think it's sufficient. Also curious to hear how you think we should handle this in the future re "We can't assume this tag will exist across Forems. Or do you mean only on DEV?"

Reply:
@benhalpern I'm wondering how you'd feel about ripping of this particular bandaid right away? We managed to avoid going down a hiring/HR feature rabbit hole on DEV and also have a classifieds section, so maybe it's time to just phase out looking for work completely?

Reply:
@citizen428 what do you think about making an Issue for this in the internal repo and putting it on the EOY Priority board in Input Needed for after this is merged? Seems like a good cleanup task to unify Forem behavior

Reply:
@mstruve Done ✅ https://github.com/forem/InternalProjectPlanning/issues/366, it's also on the board in "input needed".